wger/gym/templates/gym/reset_user_password.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% load wger_extras %}
```

Comment:
Since this feature was started, I have introduced a breadcrumb navigation for pages like this, after you add it, you can remove the link on the side.

```
{% load django_bootstrap_breadcrumbs %}


{% block breadcrumbs %}
    {{ block.super }}

    {% if perms.gym.manage_gyms %}
        {% breadcrumb "Gyms" "gym:gym:list" %}
    {% endif %}
    {% breadcrumb_raw mod_user.userprofile.gym "gym:gym:user-list" mod_user.userprofile.gym.pk %}
    {% breadcrumb_raw mod_user|format_username "core:user:overview" mod_user.pk %}
    {% breadcrumb "User" "gym:document:list" mod_user.pk %}
{% endblock %}
```